logic/src/jvmMain/kotlin/com/wire/kalium/logic/feature/call/CallManagerImpl.kt

```diff
@@ -9,6 +9,8 @@ import com.wire.kalium.logic.kaliumLogger
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.asStateFlow
 
+val calls = MutableStateFlow(listOf<Call>())
```

Comment:
How can these `calls` be static?

What if there are two users logged in?

Reply:
That's right, I missed that point